expected/epochprop.out

```diff
@@ -16,93 +16,93 @@ FROM (
   269.4742714391 |    4.4072939987 |  543.624 | -791.442 |  10235.412 | -110.450
 (1 row)
 
-SELECT 
+SELECT
        to_char(DEGREES(tp[1]), '999D9999999999'),
        to_char(DEGREES(tp[2]), '999D9999999999'),
        to_char(tp[3], '999D999'),
        to_char(DEGREES(tp[4])*3.6e6, '999D999'),
        to_char(DEGREES(tp[5])*3.6e6, '99999D999'),
        to_char(tp[6], '999D999')
 FROM (
-       SELECT epoch_prop(spoint(radians(269.45207695), radians(4.693364966)), 
+       SELECT epoch_prop(spoint(radians(269.45207695), radians(4.693364966)),
                0,
                RADIANS(-801.551/3.6e6), RADIANS(10362/3.6e6), -110,
                -100) AS tp) AS q;
-     to_char     |     to_char     | to_char | to_char  |  to_char   | to_char 
------------------+-----------------+---------+----------+------------+---------
-  269.4744079540 |    4.4055337210 |         | -801.210 |  10361.762 | 
+     to_char     |     to_char     | to_char  | to_char  |  to_char   | to_char  
+-----------------+-----------------+----------+----------+------------+----------
+  269.4744079540 |    4.4055337210 |     .000 | -801.210 |  10361.762 | -###.###
```

Comment:
Meant to ask before: Is the "-###.###" intentional? If not, it seems to me you should change '999D999' on line 25 to increase the space for the digits to get a real answer.